### PR TITLE
dolphin: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- Extend examples - use better placeholders
- Fix descriptions

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 21.12.3

### CODE REVIEW

File: pages/linux/dolphin.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-`dolphin --select {{path/to/file_or_directory}}`
+`dolphin --select {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
 
-- Launch Dolphin in a separated window:
+- Open a new window:
 
 `dolphin --new-window`
 
-- Launch Dolphin in split view:
+- Open in a split view:
```
Comment (kbdharun) [inline pages/linux/dolphin.md:22]:
```suggestion
- Open directories in split view:
```
Would this be better?

**Review by kbdharun — APPROVED** [review]

File: pages/linux/dolphin.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 `dolphin --new-window`
 
-- Launch Dolphin in split view:
+- Open specific directories in split view:
 
-`dolphin --split`
+`dolphin --split {{path/to/directory1}} {{path/to/directory2}}`
 
-- Start the Dolphin daemon (only required to use the DBus interface):
+- Launch the daemon (only required to use the DBus interface):
```
Comment (navarroaxel) [inline pages/linux/dolphin.md:26]: I prefer launch for GUI apps
```suggestion
- Start the daemon (only required to use the DBus interface):
```

File: pages/linux/dolphin.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
 # dolphin
 
-> KDE file manager.
+> Manage files and directories.
```
Comment (navarroaxel) [inline pages/linux/dolphin.md:3]:
```suggestion
> Manage files and directories by KDE.
```

Comment (kbdharun) [reply pages/linux/dolphin.md:3]: How about this?
```suggestion
> Application by KDE to manage files and directories.
```
or
```suggestion
> KDE's File manager to manage files and directories.
```

**Review by blueskyson — APPROVED** [review]

File: pages/linux/dolphin.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/linux/dolphin.md:26]: I got your point. I am going to think about how to standardize descriptions better across all pages. Maybe we can use your preference. 🤔

**Pull request merged**